docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -11,20 +11,58 @@ Ischemic heart disease (IHD) is a non-communicable cardiovascular disease which
+GBD 2017 models IHD using Myocardial infarction (MI) sequelae to estimate the prevalence of IHD, due to the challenges of disease detection and varying symptoms across the population.
+
+
+1. Acute myocardial infarction (MI): Definite and possible MI according to the third universal
+definition of myocardial infarction:
+
+a. When there is clinical evidence of myocardial necrosis in a clinical setting consistent with
+myocardial ischemia or
+b. Detection of a rise and/or fall of cardiac biomarker values and with at least one of the following: 
+  i. symptoms of ischaemia, 
+  ii. new or presumed new ST-segment-T wave changes or new left bundle branch block, 
+  iii. development of pathological Q waves in the 277 ECG, 
+  iv. imaging evidence of new loss of viable myocardium or new regional wall motion abnormality, or 
+  v. identification of an intracoronary thrombus by angiography or autopsy.
+c. Sudden (abrupt) unexplained cardiac death, involving cardiac arrest or no evidence of a noncoronary cause of death
+d. Prevalent MI is considered to last from the onset of the event to 28 days after the event and is divided into an acute phase (0–2 days) and subacute (3–28 days).
+
+2. Chronic IHD
+a. Angina; clinically diagnosed stable exertional angina pectoris or definite angina pectoris
+according to the Rose Angina Questionnaire, physician diagnosis, or taking nitrate
+medication for the relief of chest pain.
+b. Asymptomatic ischemic heart disease following myocardial infarction; survival to 28 days
+following incident MI. The GBD study does not use estimations.
+
+A systematic review for myocardial infarction was not performed for GBD 2017. Updates to systematic
+reviews are performed on an ongoing schedule across all GBD causes; an update for myocardial infarction
+will be performed in the next one to two iterations.
+A systematic review was done for myocardial infarction for GBD 2015. The search strings used were
+extensive; a full list will be provided on request.
+The dates of the search were 1/1/2009 に 2/3/2015. 38,522 studies were returned; 194 were extracted
+(this number includes extractions that were done for STEMI/NSTEMI models and revascularisation models
+that are not currently part of the MI modelling process but may be in the future).
+A systematic review for myocardial infarction was also done for GBD 2013. The extensive search terms for
+that review will be provided on request. 
+
+Vital registration, verbal autopsy data were used to model ischaemic heart disease in GBD 2017. The model outliered verbal
+autopsy data in countries and subnational locations where high-quality vital registration data were also
+available. The model also outliered non-representative subnational verbal autopsy data points, ICD8 and ICD9
+BTL data points which were inconsistent with the rest of the data and created implausible time trends,
+and data in a number of Indian states identified by experts as poor-quality. 
```

Comment:
Is this for the fatal, or nonfatal model?

Reply:
I think you could list, for the fatal and nonfatal models each, the type of data they used {epi data from a systematic review, vital registration, and verbal autopsy data, (other?)} without the outliering details! Though perhaps it's worth noting (either here or in the limitations section) that GBD didn't trust their data for india for their (nonfatal?) model

Reply:
Thanks for the feedback! I've updated this section. Whenever you have a chance, can you please take a look if my update works? Thank you!